src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultWorkspaceSemanticTokensRefreshPublisher.cs

```diff
@@ -14,12 +14,19 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer;
 internal class DefaultWorkspaceSemanticTokensRefreshPublisher : WorkspaceSemanticTokensRefreshPublisher, IDisposable
 {
     private const string WorkspaceSemanticTokensRefreshKey = "WorkspaceSemanticTokensRefresh";
+    private static readonly TimeSpan s_debounceTimeSpan = TimeSpan.FromMilliseconds(250);
```

Comment:
❓ why did this move? (I'm definitely okay with it, just curious)

Reply:
Felt odd for the fields to go from const, to read-only instance, to static, then back to instance again. I like const and statics at the top 🤷‍♂️